.github/workflows/review.yml

```diff
@@ -24,7 +26,7 @@ jobs:
       - name: 'Add ubuntu repository and install dependencies'
         run: tests/install_ubuntu_packages.sh ${{ matrix.gcc-version }}
       - name: 'Generate compiling configurations'
-        run: ./autogen.sh && ./configure --enable-test --enable-library
+        run: . tests/export_build_flags.sh ${{ env.cflags }} ${{ env.CC }}
```

Comment:
script is `export_build_flags.sh` but it runs autogen and configure.
Can we change the name to "build_configure.sh" or something like that?